### ISSUE TITLE
Documentation: fix hugo build, add missing 0.8.0 and update documentation

### ISSUE THREAD

**Issue by brokenpip3** [jenkinsci/kubernetes-operator#977]
## Broken

- [ ] The current theme is broken with the newer version of hugo
- [ ] the 0.8.0 is there but not showing up

## Missing

- [ ] CONTRIBUTING.md needs an update:
   - no minikube
   - use kind
   - use pre-commit
   - use devcontainers or flake for the dependencies

- [ ] Well know issue that needs to be documented:
  - any sso config will need a robot user so the operator can login into jenkins
  - ??? (check the issues for more)

**Comment by brokenpip3** [comment]
The website build has been fixed, now the v0.8.x section is present.
However updating hugo will require to rewrite scss part of the theme and I'm lacking of css/frontend experience, if someone from the community wants to try please let me know here, will be nice to update hugo, theme and configuration.

**Comment by userAdityaa** [comment]
Hello, Where is the scss file that needs to be updated. Can you please attach the file here. So, that I can be of help. Thankyou

**Comment by brokenpip3** [comment]
hei @userAdityaa thanks for reaching and offer help :)

the scss files are here: https://github.com/jenkinsci/kubernetes-operator/tree/master/website/assets/scss however with the current (old) version of hugo are working fine, only if we upgrade hugo we are going to see the issues.

Do you want me to create a branch with the hugo changes that are needed to led to the scss errors so you can take a look?
thanks!

**Comment by userAdityaa** [comment]
Yes, please if you can create it, will be of help to me. Thank you
